README.md

```diff
@@ -68,10 +65,17 @@ The expected format of the service account key is a **json** with the following
 }
 ```
 
-3. Configure the service account key and private key for authentication in the SDK by following one of the alternatives below:
-   - setting the fiels in the provider block: `service_account_key` or `service_account_key_path`, `private_key` or `private_key_path`
-   - setting environment variables: `STACKIT_SERVICE_ACCOUNT_KEY_PATH` and `STACKIT_PRIVATE_KEY_PATH`
-   - setting `STACKIT_SERVICE_ACCOUNT_KEY_PATH` and `STACKIT_PRIVATE_KEY_PATH` in the credentials file (see above)
+3. Configure the service account key for authentication in the SDK by following one of the alternatives below:
+
+   - setting the fiels in the provider block: `service_account_key` or `service_account_key_path`
+   - setting the environment variable: `STACKIT_SERVICE_ACCOUNT_KEY_PATH`
+   - setting `STACKIT_SERVICE_ACCOUNT_KEY_PATH` in the credentials file (see above)
+
+> **If you have provided your own RSA key-pair when creating the service account key**, you also need to configure it (takes precedence over the private key included in the service account key, if present):
+>
+> - setting the fiels in the provider block: `private_key` or `private_key_path`
+> - setting the environment variable: `STACKIT_PRIVATE_KEY_PATH`
+> - setting `STACKIT_PRIVATE_KEY_PATH` in the credentials file (see above)

```

Comment:
```suggestion
> **If you have provided your own RSA key-pair when creating the service account key**, you also need to configure your private key (takes precedence over the private key included in the service account key, if present):
>
> - setting the field in the provider block: `private_key` or `private_key_path`
> - setting the environment variable: `STACKIT_PRIVATE_KEY_PATH`
> - setting `STACKIT_PRIVATE_KEY_PATH` in the credentials file (see above)

```

Reply:
This doesn't seem to show what changed, but I suggested:
- `you also need to configure it` --> `you also need to configure your private key`
- `setting the fiels` --> `setting the field`